persistence/module.go

```diff
@@ -41,9 +41,9 @@ type persistenceModule struct {
 	// IMPORTANT: It doubles as the data store for the transaction tree in state tree set.
 	txIndexer indexer.TxIndexer
 
-	// stateTrees manages all of the merkle trees maintained by the
+	// treeStore manages all of the merkle trees maintained by the
 	// persistence module that roll up into the state commitment.
-	stateTrees modules.TreeStoreModule
+	treeStore modules.TreeStoreModule
```

Comment:
Consistent with https://github.com/pokt-network/pocket/pull/896#discussion_r1262253642, the persistence module should avoid holding a reference and retrieve it from the bus when needed.